google/cloud/documentai_toolbox/converters/config/bbox_conversion.py

```diff
@@ -109,7 +108,11 @@ def _convert_to_pixels(x: float, conversion_rate: float) -> float:
 
 
 def _convert_bbox_units(
-    coordinate, input_bbox_units, width=None, height=None, multiplier=1
+    coordinate: float,
+    input_bbox_units: str,
```

Comment:
Can `input_bbox_units` be an enum instead of `str`?

Reply:
I agree that would make more sense. However, since this value is provided by the user in a JSON file, I wonder if there is a straightforward way to make that conversion? (Besides changing the config value to be an integer) Or maybe if it would be better to use constants?